### ISSUE TITLE
Purchasing virtual products without shipping address doesn't work using epay payment window

### ISSUE THREAD

**Issue by boxyman** [bambora/checkout-magento-v2.x#20]
When I try to purchase a virtual product and therefore there are no shipping address I cannot complete the purchase. I get redirected from the checkout step with "Place order now" to the cart page.

I am using Magento 2.2.

**Comment by AllanWLie** [comment]
Hi @boxyman

The issue have been fixed in the newest release (1.3.1) https://github.com/bambora/checkout-magento-v2.x/releases/latest

Due to pending marked release you will have to follow the upgrade guide in the version changelog

**Comment by boxyman** [comment]
This issue has not been fixed. It works if you purchase a regular product. But not a virtual product.

I it helps, here are two calls with responses I get when purchasing a **virtual products**. When entering the checkout the shippingm address step is skipped because it is a virtual product. I suspect that is the problem.

```
Status code: 200
https://domain.com/rest/dansk/V1/guest-carts/xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx/payment-information

{  
   "cartId":"xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx",
   "billingAddress":{  
      "countryId":"DK",
      "region":"",
      "street":[  
         "Some street 123"
      ],
      "company":"",
      "telephone":"12345678",
      "postcode":"2000",
      "city":"København",
      "firstname":"Kaj",
      "lastname":"Kajsen",
      "saveInAddressBook":1,
      "custom_attributes":{  

      }
   },
   "paymentMethod":{  
      "method":"bambora_epay",
      "po_number":null,
      "additional_data":null,
      "extension_attributes":{  
         "agreement_ids":[  
            "1"
         ]
      }
   },
   "email":"kaj@kajsen.dk"
}
```

Here is an example of a regular product:

```
Status code: 500
https://domain.com/bambora/epay/checkout/?_=xxxxxxxxxxxxx

Status code: 200
https://domain.com/rest/dansk/V1/guest-carts/xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx/payment-information

{  
   "cartId":"xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx",
   "billingAddress":{  
      "countryId":"DK",
      "region":"",
      "street":[  
         "Some street 123"
      ],
      "company":"",
      "telephone":"12345678",
      "postcode":"2000",
      "city":"København",
      "firstname":"Kaj",
      "lastname":"Kajsen",
      "custom_attributes":{  

      },
      "saveInAddressBook":null
   },
   "paymentMethod":{  
      "method":"bambora_epay",
      "po_number":null,
      "additional_data":null,
      "extension_attributes":{  
         "agreement_ids":[  
            "1"
         ]
      }
   },
   "email":"kaj@Kajsen.dk"
}

Status code: 200
https://192.168.0.8/bambora/epay/checkout/?_=xxxxxxxxxxxxx
```
